examples/download_motions.py

```diff
@@ -23,22 +23,32 @@ def read_config(props_path: str) -> dict:
 
 # Read in your ip, username, & password
 #   (NB! you'll likely have to create this file. See tests/test_camera.py for details on structure)
-config = read_config('../secrets.cfg')
+config = read_config('camera.cfg')
 
 ip = config.get('camera', 'ip')
 un = config.get('camera', 'username')
 pw = config.get('camera', 'password')
 
 # Connect to camera
-cam = Camera(ip, un, pw)
+cam = Camera(ip, un, pw, https=True)
 
-start = (dt.now() - timedelta(hours=1))
+start = dt.combine(dt.now(), dt.min.time())
 end = dt.now()
 # Collect motion events between these timestamps for substream
-processed_motions = cam.get_motion_files(start=start, end=end, streamtype='sub')
+processed_motions = cam.get_motion_files(start=start, end=end, streamtype='main', channel=0)
+processed_motions += cam.get_motion_files(start=start, end=end, streamtype='main', channel=1)
 
-dl_dir = os.path.join(os.path.expanduser('~'), 'Downloads')
+start = dt.now() - timedelta(days=1)
+end = dt.combine(start, dt.max.time())
+processed_motions += cam.get_motion_files(start=start, end=end, streamtype='main', channel=1)
```

Comment:
Line 41-43 seems a bit superfluous to me.

why do you need to re-assign `start` and `end` here. can't we just set `start` to 1 day ago instead of start of today? Same with `end`, can't we just set it from the beginning to `dt.now()`?

Reply:
My camera (maybe others, but my for sure) does not support searches that span more than a single day.
So the first search is for "today", and the second one for "yesterday", which is how I make sure to cover > 24h and see all videos.

Reply:
ah okay, that makes sense! Thank you

Reply:
My current use case is every day, review the videos of the past day.
The web app and mobile apps expose a calendar, but I don't think it works all that great. So longer term maybe we want a GUI to select what date range to look for (and I can make it default to whatever I like).